src/Features/CSharp/Portable/Completion/KeywordRecommenders/VoidKeywordRecommender.cs

```diff
@@ -48,7 +48,7 @@ protected override bool IsValidContext(int position, CSharpSyntaxContext context
                 IsUnsafeCastTypeContext(context) ||
                 IsUnsafeDefaultExpressionContext(context) ||
                 context.IsFixedVariableDeclarationContext ||
-                context.SyntaxTree.IsLocalFunctionDeclarationContext(position, cancellationToken);
+                context.SyntaxTree.IsLocalFunctionDeclarationContext(position, SyntaxKindSet.AllLocalFunctionModifiers, cancellationToken);
         }
 
         private bool IsUnsafeDefaultExpressionContext(CSharpSyntaxContext context)
```

Comment:
what about tests for thins like primitive types.  i would expect to have `int` offered after something like `{ [Foo] $$`